src/ontology/mondo-edit.obo

```diff
@@ -430935,6 +431179,14 @@ xref: ICD10:H31.0 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:58
 xref: Orphanet:580951 {source="MONDO:equivalentTo"}
 is_a: MONDO:0005328 {source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"} ! eye disorder
 
+[Term]
+id: MONDO:0035586
+name: Cramp-fasciculation syndrome
+subset: ordo_disorder
+xref: ICD10:G90.8 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:581271"}
+xref: Orphanet:581271 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005218 {source="https://orcid.org/0000-0001-5208-3432"} ! muscular disorder
```

Comment:
FRom https://rarediseases.info.nih.gov/diseases/6205/cramp-fasciculation-syndrome (GARD):
Cramp-fasciculation syndrome (CFS) is a rare condition of the muscles characterized by persistent muscle cramping and twitching ([fasciculations](http://www.nlm.nih.gov/medlineplus/ency/article/003296.htm)) in otherwise healthy individuals. This can lead to muscle discomfort, pain, or tiredness. Muscles in the leg are most commonly affected, although this condition may involve several parts of the body. Symptoms are thought to be due to over-activity of the associated nerves. In most cases, CFS occurs [sporadically](http://www.ncbi.nlm.nih.gov/books/NBK5191/def-item/sporadic/) in people with no family history of the condition. There is limited information about the treatment of CFS, but certain medications have been reported as beneficial in individual cases.

I think that the muscle part is a symptom, and therefore I don't think we can say with certainty that this disease is a muscular disorder. 
I think a better parent should be "syndromic disorder"

Reply:
+1